docker-images/kafka-based/kafka/scripts/kafka_connect_run.sh

```diff
@@ -37,15 +37,15 @@ fi
 # directory avoids trying to create it (and logging a permission denied error)
 export LOG_DIR="$KAFKA_HOME"
 
-# enabling Prometheus JMX exporter as Java agent
+# Enable JMX Prometheus Exporter as Java agent
```

Comment:
aren't we going in the opposite direction so "Prometheus JMX exporter" as it was before?